packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -59,6 +59,23 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
         };
 
         meeting.mediaProperties.webrtcMediaConnection.roapMessageReceived(roapMessage);
+
+        let mediaServer;
+
+        // Attempt to collect the media server from the roap message.
+        try {
+          mediaServer = roapMessage.sdp
```

Comment:
I've checked and unfortunately looks like the browser is removing the username from the origin line in the remote SDP, so we cannot read it later from the test and we have to parse it out from the incoming message like you're doing it here. So for now this code needs to stay as it is here, but can you please wrap it in a function like `extractMediaServer()` or something like this and just call the function from here, so it's nicely contained and easier to remove in the future.